Build failure for `samples/standalone` on Gradle 4.6

```
$ gradle nbm
...
FAILURE: Build failed with an exception.

* Where:
Build file '/Users/eron/Projects/gradle-nbm-plugin/samples/standalone/build.gradle' line: 17

* What went wrong:
A problem occurred evaluating root project 'standalone'.
> Failed to apply plugin [id 'cz.kubacki.nbm']
   > Cannot add a configuration with name 'implementation' as a configuration with that name already exists.
```

```
$ gradle --version
------------------------------------------------------------
Gradle 4.6
------------------------------------------------------------

Build time:   2018-02-28 13:36:36 UTC
Revision:     8fa6ce7945b640e6168488e4417f9bb96e4ab46c

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          1.8.0_172 (Oracle Corporation 25.172-b11)
OS:           Mac OS X 10.13.4 x86_64
```

--- a/samples/standalone/build.gradle
+++ b/samples/standalone/build.gradle
@@ -9,12 +9,16 @@ buildscript {
         // }
     }
     dependencies {
-        classpath "cz.kubacki.gradle.plugins:gradle-nbm-plugin:1.10.0"
+        classpath "cz.kubacki.gradle.plugins:gradle-nbm-plugin:1.17.0"
         gradleApi()
     }
 }
 apply plugin: 'java'
 apply plugin: 'cz.kubacki.nbm'
+
+ext {
+    netbeansVersion = "RELEASE82"
+}
 
 repositories {
     mavenCentral()
@@ -35,12 +39,12 @@ nbm {
 }
 
 dependencies {
-    compile 'org.netbeans.api:org-openide-awt:RELEASE74'
-    compile 'org.netbeans.api:org-openide-util:RELEASE74'
-    compile 'org.netbeans.api:org-netbeans-api-annotations-common:RELEASE74'
+    compile "org.netbeans.api:org-openide-awt:$netbeansVersion"
+    compile "org.netbeans.api:org-openide-util:$netbeansVersion"
+    compile "org.netbeans.api:org-netbeans-api-annotations-common:$netbeansVersion"
 
     // Use 'implementation' configured if implementation dependency is needed for some module
-    // implementation 'org.netbeans.api:org-netbeans-api-annotations-common:RELEASE74'
+    // implementation "org.netbeans.api:org-netbeans-api-annotations-common:$netbeansVersion"
 
     // example of dependency that will be copied to modules/ext and added to ClassPath: attribute in MANIFEST.MF
     compile 'org.slf4j:slf4j-api:1.7.2'

